internal/cluster/distribution/eks/eksprovider/workflow/save_cluster_version_activity.go

```diff
@@ -0,0 +1,54 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package workflow
+
+import (
+	"context"
+
+	"github.com/banzaicloud/pipeline/internal/cluster/distribution/eks/eksmodel"
+)
+
+const SaveClusterVersionActivityName = "eks-save-cluster-version"
```

Comment:
**Question**: is `save` the correct terminology here?

On opinionated ground I **MAY** propose `persist` or `record` here, because saving intuitively felt like something used for backup/restore or other failsafe mechanisms, but in reality it seems like only an administrational step for an information source.

Feel free to correct me if I'm misreading the nomenclature here.

Reply:
I used `save` because that was used across the activities here, like `eks-save-network-details` and `eks-save-node-pools`, I think we need to change them as well in this case.

Reply:
You are right, those examples indicate we should conventionally use save here.

It is a discussion separate from this PR to decide whether we want to change the corresponding terminology.